From 94ea3cd9ec1e59071a29ca6ed33386bf50c6355d Mon Sep 17 00:00:00 2001
From: Dylan Van Assche <me@dylanvanassche.be>
Date: Fri, 27 Nov 2020 20:08:47 +0100
Subject: [PATCH] broadband-modem: Do not send ATH during probing

ATH (AT hangup) rejects an incoming call during probing. This is a hack,
needs improvement, but suggested by the maintainer: https://gitlab.freedesktop.org/mobile-broadband/ModemManager/-/issues/194
---
 src/mm-broadband-modem.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mm-broadband-modem.c b/src/mm-broadband-modem.c
index 97c60b68..9e55b011 100644
--- a/src/mm-broadband-modem.c
+++ b/src/mm-broadband-modem.c
@@ -7607,7 +7607,7 @@ modem_voice_check_support (MMIfaceModemVoice *self,
 
     /* Check ATH support */
     mm_base_modem_at_command (MM_BASE_MODEM (self),
-                              "H",
+                              "I",
                               3,
                               FALSE,
                               (GAsyncReadyCallback)ath_format_check_ready,
-- 
2.25.1

